portfolio/src/main/webapp/script.js

```diff
@@ -127,3 +127,23 @@ const slideToggle = (elementId, duration)  => {
     slideUp(elementId, duration);
   }
 }
+
+/* fetches comments content from webserver and adds to DOM in container with id elementID */
+const addToDOM = (elementId) => {
+  fetch("/comments").then(response => response.json()).then((comments) => {
+
+    // add each parsed json.commentText to the DOM
+    const destinationDiv = document.getElementById(elementId);
+    
+    for (comment in comments) {
+      destinationDiv.appendChild(createSpan(comments[comment].commentText));
+    }
+  });
+}
+
+/* creates a span element to be added to the DOM containing text 'text' */
+const createSpan = (text) => {
```

Comment:
Maybe rename to something like `createCommentElement()`? The implementation doesn't need to be a span, per se, and it'd be annoying to have to change the name and function documentation just because of the template.

Reply:
done!